app_dart/test/service/luci_build_service_test.dart

```diff
@@ -740,12 +740,11 @@ void main() {
 
     test('Reschedule an existing build', () async {
       when(mockBuildBucketClient.scheduleBuild(any)).thenAnswer((_) async => generateBuild(1));
-      final bool rescheduled = await service.rescheduleBuild(
+      await service.rescheduleBuild(
```

Comment:
I would add validation of the new return object here.

Reply:
Sounds good. I'll see what I can do.